app/views/idv/gpo_verify/index.html.erb

```diff
@@ -9,6 +9,18 @@
 
 <% title t('forms.verify_profile.title') %>
 
+<%= render AlertComponent.new(type: :info, class: 'margin-bottom-4', text_tag: 'div') do %>
+  <p>
+    <%= t('forms.verify_profile.alert_info') %>
+    <br>
+    <%= render 'shared/address', address: @gpo_verify_form.pii %>
```

Comment:
Is there ever a case where `pii` might not be present? I can't imagine it should be blank.

Reply:
good call @aduth 👍🏿 ... i added a check to ensure it's not nil before rending the address partial

Reply:
I don't necessarily want to suggest being overly guarded if it's not actually going to be blank in real-world usage, but I'm not sure if it ever would be. Maybe it'd depend if we could devise a spec where the condition would be necessary, ideally without resorting to manipulating the record directly with code (i.e. only a situation a real user could find themselves in).